spec/ldpath_program_spec.rb

```diff
@@ -38,7 +39,10 @@
     end
 
     it "should work" do
+      b1 = RDF::Node.new('b1')
       graph << [object, RDF::Vocab::DC.title, "Hello, world!"]
+      graph << [object, RDF::Vocab::DC.title, b1]
```

Comment:
Adding in a blank node to the title because there needs to be something other than string for test differentiation.